src/utils/mailer.js

```diff
@@ -0,0 +1,36 @@
+const dotenv = require('dotenv');
+
```

Comment:
can we move the require statements after each other and after it a new line and then `dotenv.config`

Reply:
done